docs/applications/nube-sdk/browser-apis.md

```
@@ -84,3 +84,46 @@ The `navigate` method is particularly useful when you need to:
+The `postMessageToIframe` method accepts two parameters:
+- The `Iframe` component instance
+- A message object containing the data you want to send
+
+```typescript title="main.ts"
+import { Iframe } from "@tiendanube/nube-sdk-jsx";
+import type { NubeSDK } from "@tiendanube/nube-sdk-types";
+
+export function App(nube: NubeSDK) {
+	const Component = (
+		<Iframe
+			src="https://tiendanube.com/"
```

markdownlint-cli2 (0.18.1)

Hard tabs
Column: 1

(MD010, no-hard-tabs)